data/troubleshooting.xml

```diff
@@ -123,13 +123,13 @@ Logging already initialized. Skipping...
                 your system is using port 8080 or 8443, the default ports that eXist's embedded web server, Jetty, uses. To see whether this is the case, enter <ulink url="http://localhost:8080/">http://localhost:8080/</ulink> in your browser. If
                 another service occupies this port, you cannot start up eXist-db unless you shut
                 down the service in question or make eXist-db use another port. To make eXist-db use
-                another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty.xml</filename> inside your eXist-db installation in a
+                another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty-http.xml</filename> inside your eXist-db installation in a
                 text or XML editor and change the value "8080" in</para>
             <synopsis format="linespecific">&lt;Set name="port"&gt;&lt;SystemProperty name="jetty.port" default="8080"/&gt;&lt;/Set&gt;</synopsis>
             <para>to a port that is not used, e.g. "8899":</para>
             <synopsis format="linespecific">&lt;Set name="port"&gt;&lt;SystemProperty name="jetty.port" default="8899"/&gt;&lt;/Set&gt;</synopsis>
             <para>eXist-db uses port 8443 for confidential communication. Another service may also
-                be using this port. To make eXist-db use another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty.xml</filename> and change the value
+                be using this port. To make eXist-db use another port, open the file <filename moreinfo="none">/tools/jetty/etc/jetty-https.xml</filename> and change the value
                 "8443" in</para>
             <synopsis format="linespecific">&lt;Set name="confidentialPort"&gt;8443&lt;/Set&gt;</synopsis>
```

Comment:
Likewise, the syntax of the line to change is still not correct - please see my previous comments.